web/maps.html

```diff
@@ -167,17 +167,17 @@
                         </div>
                         <div id='tools-detail-buildings-floors' class='hidden'>
                             Floor: <select id='tools-detail-buildings-floor-list'></select>
-                            <input type='button' value='...'>
+                            <input type='button' value='...' onclick="SCENERY_EDITOR.openTexturePreviewModal('tools-detail-buildings-floor-list')">
```

Comment:
Not what I meant. I meant a selection modal that shows all the textures available in a list and lets the user choose which one they want, just like the model view.

Reply:
Something like this:

https://imgur.com/a/oKPvmLB

Reply:
Gotcha